doc/base/classes.xml

```diff
@@ -20,7 +20,14 @@
 			<argument index="3" name="a8" type="int">
 			</argument>
 			<description>
-				Make a color from red, green, blue and alpha. Arguments can range from 0 to 255.
+				Returns a color made up of red, green, blue and alpha components. Arguments can range from 0 to 255.
```

Comment:
Avoid passive voice.
`Returns a color with red, green, blue and alpha components. Each color channel has 8bit of information: it ranges from 0 to 255`

Note: Color8 is ambiguous, so it's worth mentioning it's 8bit per channel. It builds a 32bit color in the end.

Writing guidelines: https://github.com/GDquest/godot-3-guides/blob/master/contribution-guides/class-reference-styleguide.md